### DISCUSSION TITLE
Number to String Type Conversion

### DISCUSSION THREAD

**Discussion by meganwolf0** [open-policy-agent/community#200] [OPA and Rego] [answered]
Hello,

I've run through the docs and this discussion forum looking for an answer to this question, so if it already exists I apologize in advance! 

I'm trying to simply convert the year and month numbers returned from the "time.date" function into a string to create a timestamp I can pass to an http request (Playground here: https://play.openpolicyagent.org/p/uo6ecUpoxn).

It doesn't seem there's a built in method to achieve this? Is there a better way than my janky solution in the playground?

**Comment by anderseknert** [comment, marked as answer]
Hi @meganwolf0 👋 

You'd normally use the `sprintf` built-in function for that:

```rego
year := 2022
year_str := sprintf("%v", [year])
```

Definitely not easy to find unless you know what to look for, so no need to apologize :)

**Reply by meganwolf0** [reply]
That worked perfect, thanks so much for the help!
